pkg/config/output_formats.go

```diff
@@ -0,0 +1,57 @@
+package config
+
+type Formats struct {
+	Text        Text         `mapstructure:"text"`
+	JSON        SimpleFormat `mapstructure:"json"`
+	Tab         Tab          `mapstructure:"tab"`
+	HTML        SimpleFormat `mapstructure:"html"`
+	Checkstyle  SimpleFormat `mapstructure:"checkstyle"`
+	CodeClimate SimpleFormat `mapstructure:"code-climate"`
+	JUnitXML    JUnitXML     `mapstructure:"junit-xml"`
+	TeamCity    SimpleFormat `mapstructure:"teamcity"`
+	Sarif       SimpleFormat `mapstructure:"sarif"`
+}
+
+func (f *Formats) IsEmpty() bool {
+	styles := []SimpleFormat{
```

Comment:
I guess we should rename the local var as well: 
```suggestion
	formats := []SimpleFormat{
```